inc/notificationtarget.class.php

```diff
@@ -677,26 +677,25 @@ public function addItemAuthor() {
 
       // members/managers of the group allowed on object entity
       // filter group with 'is_assign' (attribute can be unset after notification)
-      $criteria = array_merge(
+      $criteria = $this->getDistinctUserCriteria() + $this->getProfileJoinCriteria();
+      $criteria['FROM'] = Group_User::getTable();
+      $criteria['INNER JOIN'] = array_merge(
          [
-            'INNER JOIN' => [
-               User::getTable() => [
-                  'ON' => [
-                     Group_User::getTable()  => 'users_id',
-                     User::getTable()        => 'id'
-                  ]
-               ],
-               Group::getTable() => [
-                  'ON' => [
-                     Group_User::getTable()  => 'groups_id',
-                     Group::getTable()       => 'id'
-                  ]
+            User::getTable() => [
```

Comment:
This join instruction has to precede the one from the getProfileJoinCriteria method which is based on glpi_users.id value.